lib/kubernetes/src/api.ts

```diff
@@ -18,8 +18,19 @@ import { K8sResource } from "./common";
 import { kubernetesError } from "./utils";
 import { log } from "@opstrace/utils";
 
+const dryRunKubeApiWrites = process.env.DRY_RUN_KUBERNETES_API_WRITES
```

Comment:
In a cluster that was normally set up -- what is the best / intended way to set this env var, and then make it take effect? Can we test this as part of CI with a little bit of a `kubectl` dance?

Reply:
I imagine you'd only ever use it when running the controller locally during manual controller testing (to see what _this version of the controller_ would change). At least that's the extent of its purpose at this point.